### PR TITLE
Feature/5547 load only specified content blocks

### PR DESCRIPTION
**Pull request by zorafuchs**

### CODE REVIEW

**Comment by lukaselmer** [comment]
what about the documentation / readme.md? can you also add an example in the index.html file?

File: src/app/main.ts [hunk 1]
```diff
@@ -33,6 +33,10 @@
     jQuery(document).on('renuo-cms-reload', function () {
       controller.init();
     });
+
+    jQuery(document).on('renuo-cms-reload', function (event:Event, htmlContentBlocks:HTMLElement[]) {
```
Comment (lukaselmer) [inline src/app/main.ts]: - i would rather prefer to only (re)load a single content block ("Reload single content via trigger")
- as a developer, i would like to call it via a string instead of an html element, where the string is the content key

File: src/app/main.ts [hunk 2]
```diff
@@ -33,6 +33,10 @@
     jQuery(document).on('renuo-cms-reload', function () {
```
Comment (lukaselmer) [inline src/app/main.ts]: here we reload all contents (also if you pass an html element / string)

File: src/app/main.ts (hunk 1 above)
Comment (lukaselmer) [inline src/app/main.ts]: and here we some more content blocks

File: src/app/controllers/view_controller.spec.ts [hunk 3]
```diff
@@ -64,4 +64,64 @@ describe('ViewController', function () {
     const drawSecondDom = drawerSpy.calls.mostRecent().args[0];
     expect(drawSecondDom.contentBlock.content).toBe(`real content ${drawSecondDom.contentBlock.contentPath}`);
   });
+
+  it('initializes the contents of specified content-blocks', function () {
```
Comment (lukaselmer) [inline src/app/controllers/view_controller.spec.ts]: it seems to me like there is a lot of duplicated code here, so i suggest to refactor it

File: src/app/main.ts (hunk 1 above)
Comment (zorafuchs) [inline src/app/main.ts]: Ahh ok, Simon said it should be possible to reload multiple and that they should already be htmlElements. Maybe you should discuss what you really need, because I don't know it...

File: src/app/main.ts (hunk 2 above)
Comment (zorafuchs) [inline src/app/main.ts]: so i should rename the new event?

File: src/app/views/services/content_block_finder.spec.ts [hunk 4]
```diff
@@ -34,4 +34,14 @@ describe('ContentBlockFinder', function () {
     expect(pairs).toContain(['other-path', 'my-key']);
     expect(pairs).toContain(['xxx-path', 'my-key']);
   });
+
+  it('finds given content block', function () {
```
Comment (rbuetzer) [inline src/app/views/services/content_block_finder.spec.ts]: I'd also test the corner case: No block found -> it returns an empty array

**Comment by rbuetzer** [comment]
Apart from that missing corner-case test: Looks very good.
